modules/channel-web/assets/inject.js

```diff
@@ -11,10 +11,15 @@ function injectDOMElement(tagName, targetSelector, options) {
 
 window.addEventListener('message', function(payload) {
   const data = payload.data
-  if (!data || !data.type || data.type !== 'setClass') {
+  if (!data || !data.type) {
     return
   }
-  document.querySelector('#bp-widget').setAttribute('class', data.value)
+
+  if (data.type === 'setClass') {
+    document.querySelector('#bp-widget').setAttribute('class', data.value)
+  } else if (data.type === 'setWidth') {
+    document.querySelector('#bp-widget').style.width = data.value
```

Comment:
Not really, because otherwise you also need to create a class before. It could be replace by setStyle, but I couldn't think of what other properties would need to be changed on the parent iframe (maybe setSize, for height and width)